app/models/ticket_purchase.rb

```diff
@@ -58,4 +63,24 @@ def self.update_quantity(conference, quantity, ticket, user)
     purchase.quantity = quantity if quantity > 0
     purchase
   end
+
+  def pay(payment)
+    self.update_attributes(paid: true, payment_id: payment.id)
+  end
+
+  protected
+
+  def create_physical_tickets
```

Comment:
But then what about Free tickets they does not have any payment and paid attribute is already true.:thinking: